### ISSUE TITLE
Only disable elements that are not already disabled (can massively speed up completion)

### ISSUE THREAD

**Issue by kristiansp** [craftcms/feed-me#1241]
### Description
When feed is set to disable elements, every single element is explicitly set to `$element->enabled = false;` and then saved, even if it already was disabled. When keeping disabled elements around, this can massively slow down the feed completion, as well as using unnecessary resources resaving a lot of elements.

A simple `if` to check if an element is already disabled, and then only save it if it is enabled, sped up my feed completion from 44.82s to 2.25s(!), although mileage of course may very. (I use Feed Me to sync content from an external source, running every hour, so most of the time there are no big changes).

Even if the gains are not so huge in every scenario, it seems a little bit like a simple no-brainer that can't really negatively impact anyone.

I already have a PR for this from August 2020:
#736

### Steps to reproduce

1. Set up feed with import strategy to disable missing elements
2. Have a substantial amount of disabled elements
3. Wait, wait and wait for the import to complete while it's disabling already disabled elements

### Additional info

- Craft version: 4.3.8.2
- PHP version: 8.0.26
- Database driver & version: MySQL 10.4.27
- Plugins & versions: FeedMe 5.0.5 (but basically any version)

**Comment by kristiansp** [comment]
I see that the old pull request has a conflict now because of extra parameters in the saveElement method. But it should suffice to go from this:
```
$element->enabled = false;
$elementsService->saveElement($element, true, true, Hash::get($this->feed, 'updateSearchIndexes'));
```
to this:
```
if ($element->enabled) {
        $element->enabled = false;
        $elementsService->saveElement($element, true, true, Hash::get($this->feed, 'updateSearchIndexes'));
}
```

**Comment by angrybrad** [comment]
Resolved in https://github.com/craftcms/feed-me/pull/1248

**Comment by brandonkelly** [comment]
Feed Me 4.6.0 (Craft 3) and 5.1.0 (Craft 4) have been released with this change (via #1248) 🎉

**Comment by kristiansp** [comment]
Amazing, thanks!
